docs/data/core-edit-post.md

```diff
@@ -0,0 +1,271 @@
+# **core/edit-post**: The Editor's UI Data
+
+## Selectors 
+
+### getEditorMode
+
+Returns the current editing mode.
+
+*Parameters*
+
+ * state: Global application state.
+
+
+### isEditorSidebarOpened
+
+Returns true if the editor sidebar is opened.
+
+*Parameters*
+
+ * state: Global application state
+
+*Returns*
+
+Whether the editor sidebar is opened.
```

Comment:
That is what it does, but what it returns is <code>\`true\` if sidebar is open; \`false\` if sidebar is not open</code>.

Reply:
>That is what it does, but what it returns is <code>\`true\` if sidebar is open; \`false\` if sidebar is not open</code>.

Aside: Typically the primary description would include a "true if ..., otherwise false". Also there is type information available in the JSDoc `{boolean}` we are not including here.